raitools/raitools/_managers/explainer_manager.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+"""Defines the Explainer Manager class."""
+
+import warnings
+from scipy.sparse import issparse
+from interpret_community.mimic.mimic_explainer import MimicExplainer
+from interpret_community.mimic.models.lightgbm_model import (
+    LGBMExplainableModel)
+from interpret_community.mimic.models.linear_model import (
+    LinearExplainableModel)
+from interpret_community.common.constants import ModelTask
+from raitools._managers.base_manager import BaseManager
+
+SparseNumFeaturesThreshold = 1000
```

Comment:
Can all constants use upper snake case? (`SPARSE_NUM_FEATURES_THRESHOLD`) Pascal case is usually reserved for classes in Python.

Reply:
done!